pyquil/paulis.py

```diff
@@ -170,6 +172,24 @@ def __radd__(self, term):
         assert isinstance(term, Number)
         return self + term
 
+    def __sub__(self, term):
+        """Subtracts a PauliTerm from this one.
+
+        :param PauliTerm term: A PauliTerm object
+        :returns: A PauliSum object representing the sum of this PauliTerm and term
+        :rtype: PauliSum
+        """
+        return self + -1. * term
+
+    def __rsub__(self, other):
+        """Subtracts this PauliTerm from a Number or PauliTerm one.
+
+        :param PauliTerm term: A PauliTerm object
+        :returns: A PauliSum object representing the sum of this PauliTerm and term
+        :rtype: PauliSum
+        """
+        return other + -1. * self
```

Comment:
@willzeng Given that you didn't comment, I assume here you agree- You should be able to just use the already defined `__sub__`. That might be clearer. Also, I don't know that it matters too much.